ext/storage_sources/gcp_store/test/test_gcp_connection.cpp

```diff
@@ -94,7 +94,7 @@ generate_test_prefix()
 
 TEST_CASE("Testing class gcpConnection", "gcp-connection")
 {
-    std::string test_bucket_name = "unit_testing_gcp";
+    std::string test_bucket_name = "gcptestext-us";
```

Comment:
From memory we delete the contents of these buckets on test finish, so having the python and unittests run on the same bucket in parallel could cause interference. Were there issues with creating a new unit_testing_gcp bucket?

Reply:
We delete all of the unit test related contents to do with the test run. We loop through all objects of the unit test through a generated prefix through the generate_test_prefix() function. 
Example of generated test prefix: "gcptest/unit/2022-31-01-16-34-10/623843294/".